modules/ivc_champva/app/services/ivc_champva/file_uploader.rb

```diff
@@ -11,49 +11,49 @@ def initialize(form_id, metadata, file_paths, attachment_ids, insert_db_row = fa
     end
 
     def handle_uploads
-      pdf_results = @attachment_ids.zip(@file_paths).map do |attachment_id, file_path|
+      results = @attachment_ids.zip(@file_paths).map do |attachment_id, file_path|
```

Comment:
The top of these changes are simply nomenclature updates since we are updating more than PDFs